6.2_2000_run.py

```diff
@@ -0,0 +1,27 @@
+"""
+Yaakov Haimoff
+"""
+import time
+
+def timer(func, *args):
+    """
+    Times how long a function takes to execute.
+
+    Parameters:
+    func (function): The function to be timed.
+    *args: Any arguments that need to be passed to the function.
+
+    Returns:
+    None
+    """
+    start_time = time.time()
+    map(func, *args)
```

Comment:
The exercise didn't want you to run the function with each parameter separately, the function needs all the arguments to run, not only one of them each time